pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -240,8 +242,53 @@ func (facts *PipelineRunFacts) GetChildReferences() []v1.ChildStatusReference {
 	return childRefs
 }
 
+func (t *ResolvedPipelineTask) getDisplayName(customRun *v1beta1.CustomRun, taskRun *v1.TaskRun, c v1.ChildStatusReference) v1.ChildStatusReference {
```

Comment:
NIT: I find the name/structure of this function slightly confusing - it does not return the display name, instead it sets it on a `ChildStatusReference` that must be passed as input. 

Since the `ChildStatusReference` is not used to build the displayName, I thought it would be better to have a `getDisplayName` function that returns a string - any particular reason for having it this way?